src/assignments/benson/lesson_6_answers.clj

```diff
@@ -0,0 +1,93 @@
+(ns assignments.benson.lesson-6-answers)
+
+(def pizza-machine {:state :start :coins []})
+
+(def price-of-pepperoni-pizza 600)
+
+(def refund {:type :refund})
+
+(defn coin-value [machine] (reduce + (:coins machine)))
+
+(defmulti handle
+  "Handler for pizza-machine events"
+  (fn [machine event] [(:state machine) (:type event)]))
+
+
+;; I couldn't add any docstrings to defmethod so I've put a 
```

Comment:
That's right.  Unfortunately, you can only add a docstring to the defmulti form.

That's because they all share the same VAR.